README.md

```diff
@@ -1,5 +1,130 @@
+
+#### De-Identification
+
+The term de-identification is used within US-based cybersecurity and data privacy concepts. The National Institutes of Standards and Technology ("NIST") relies on this concept for removing identifiers from personal information, defining de-identification as:
+
+> general term for any process of removing the association between a set of identifying data and the data subject [2].
+
+By removing this association between the identifier and the information, information becomes de-identified, thereby reducing certian privacy concerns. Additional uses of the term "de-idenfitication" can be found in the Health Insurance Portability and Accountability Act (HIPAA) and the California Consumer Privacy Act of 2018. 
+
+Based on these sources, our project generated the following generalized definition for "de-identification":
+
+> Information that cannot reasonably identify, relate to, describe, be capable of being associated with, or be linked, directly or indirectly, to a particular individual.
+
+#### Anonymization
+
+The concept of annonymization is used across a variety of different legal frameworks and regions. For example, NIST defines this term as:
+
+> the process that removes the association between the identifying dataset and the data subject [2]. 
+
+Further, while the term annonymization is not defined within the GDPR, it is used by the European Data Protection Board ("EDPB") in further explaining the impact of the GDPR within the context of geolocation tracking. [3] The Guidelines state that
+
+> Anonymisation refers to the use of a set of techniques in order to remove the ability to link the data with an identified or identifiable natural person against any “reasonable” effort. This “reasonability test” must take into account both objective aspects (time, technical means) and contextual elements that may vary case by case (rarity of a phenomenon including population density, nature and volume of data). If the data fails to pass this test, then it has not been anonymised and therefore remains in the scope of the GDPR. ¶ 15
+
+Futher, these Guidelines outline three factors to determine the "robustmenss of anonymization": 
+
+> (i) singling-out (isolating an individual in a larger group based on the data); (ii) linkability (linking together two records concerning the same individual); and (iii) inference (deducing, with significant probability, unknown information about an individual). ¶ 16
+
+Based on this guidance, our project generated the following generalized defintion for "anonymization":
+
+> A process that removes the association between the identifying dataset and the data subject.
+
+#### Aggregation
+
+The concept of aggregated data bridges both the annonymity of information and the amount of information collected. In essence, it the combination of volume plus annonymity that allows aggregated data to be less prone to privacy concerns. The CCPA directly recognizes the concept of aggregation as follows:
+
```

Comment:
STRONGEST

Reply:
note:  SOME research found that aggregated data can be not private, but this is highly unusual. for example, average weight of 99 people = 200 pounds.   Add a 100th person, and average weight = 201 pounds = (200*100)/100 + 100/100.  Therefore, that person is 200+100 = 300 pounds.

Reply:
@mjs624 do you have a citation for this?